commander/src/utils/core/cache.ts

```diff
@@ -67,24 +67,18 @@ const stopCommand = async (
 	network: NETWORK,
 	name: string,
 ): Promise<string> => {
-	try {
-		const {
-			components: {
-				cache: { password },
-			},
-		}: LiskConfig = await getLiskConfig(installDir, network);
-		const { redisPort } = (await describeApplication(
-			name,
-		)) as PM2ProcessInstance;
-
-		if (password) {
-			return `${REDIS_CLI} -p ${redisPort} -a ${password} shutdown`;
-		}
+	const config = await getLiskConfig(installDir, network);
+	const password = config?.components?.cache?.password;
+	if (password === undefined) {
+		throw new Error('Config password is not set.');
+	}
```

Comment:
why do we need to throw here? let's say on alphanet the cache config is 
```
"cache": {
    "enabled": true
}
```
so this case the `password` will result in `undefined`, but that's the expected behavior 
this is the correct behaviour
```
if (password) {
		return `${REDIS_CLI} -p ${redisPort} -a ${password} shutdown`;
	}

	return `${REDIS_CLI} -p ${redisPort} shutdown`;
```